package.json

```diff
@@ -67,6 +67,7 @@
     "eslint-plugin-react-hooks": "^4.5.0",
     "husky": "^7.0.4",
     "jest": "^27.4.1",
+    "jest-styled-components": "^7.0.8",
```

Comment:
https://github.com/styled-components/jest-styled-components

^^ I needed this library to test styled-components based on components' styles.